### PR TITLE
New package: qomui-0.8.1

### PR DESCRIPTION
**Pull request by youngjinpark20**

### CODE REVIEW

**Comment by youngjinpark20** [comment]
I'm not sure if I should add a runit service because this by default only has systemd

File: srcpkgs/qomui/template [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# Template file for 'qomui'
+pkgname=qomui
+version=0.8.1
+revision=1
```
Comment (maxice8) [inline srcpkgs/qomui/template]: use the python3-module build_style instead of defining  do_build and do_install

**Comment by youngjinpark20** [comment]
Didn't know that was an option.... I feel stupid

File: srcpkgs/qomui/template [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+build_style=python3-module
+hostmakedepends="python3 python3-setuptools"
+depends="
+	python python3 python3-PyQt5 python3-dbus python3-psutil python3-requests \
```
Comment (maxice8) [inline srcpkgs/qomui/template]: don't skip the first line and use whitespace, not tabs, for identation on following lines.

also don't need to use `\` inside quoted variables

**Pull request closed without merging**